README.md

```diff
@@ -11,9 +11,9 @@
 
 Parity Fether aims to be the lightest and simplest decentralized wallet. It supports Ether and ERC-20 tokens, and runs on top of [Parity Ethereum](https://github.com/paritytech/parity-ethereum) light client. This allows smooth synchronization and interaction with the Ethereum blockchain, in a decentralized manner.
 
-By default, Parity Fether alpha runs on the Kovan test network. You can receive free Kovan Ether by posting your address in the [Kovan Faucet](https://gitter.im/kovan-testnet/faucet) Gitter channel. Fether will download and launch Parity Ethereum node at startup if it's not found on the computer. You can also separately launch your Ethereum client, Fether will automatically connect to it.
+By default, Fether will launch its embedded Parity Ethereum light client. You can also separately launch your Ethereum client (e.g on a [testnet](https://wiki.parity.io/Fether-FAQ#how-to-launch-fether-on-a-different-network)), Fether will automatically connect to it.
```

Comment:
Instead of `You can also separately launch your Ethereum client (e.g on a [testnet](https://wiki.parity.io/Fether-FAQ#how-to-launch-fether-on-a-different-network)), Fether will automatically connect to it.`, suggest changing it to `You can also separately launch your Ethereum client and Fether will automatically connect to it, as described in the [Fether FAQ](https://wiki.parity.io/Fether-FAQ#how-to-launch-fether-on-a-different-network).`